packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts

```diff
@@ -1,50 +1,80 @@
-import { html, ref } from "@microsoft/fast-element";
+import { html, ref, ViewTemplate } from "@microsoft/fast-element";
 import { endTemplate, startTemplate } from "../patterns/start-end";
 import { AccordionItem } from "./accordion-item";
 
+const expandedSVG = html<AccordionItem>`
+    <svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+        <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M3.78 3h12.44c.43 0 .78.35.78.78v12.44c0 .43-.35.78-.78.78H3.78a.78.78 0 01-.78-.78V3.78c0-.43.35-.78.78-.78zm12.44-1H3.78C2.8 2 2 2.8 2 3.78v12.44C2 17.2 2.8 18 3.78 18h12.44c.98 0 1.78-.8 1.78-1.78V3.78C18 2.8 17.2 2 16.22 2zM14 9H6v2h8V9z"
+        />
+    </svg>
+`;
+
+const collapsedSVG = html<AccordionItem>`
+    <svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+        <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M16.22 3H3.78a.78.78 0 00-.78.78v12.44c0 .43.35.78.78.78h12.44c.43 0 .78-.35.78-.78V3.78a.78.78 0 00-.78-.78zM3.78 2h12.44C17.2 2 18 2.8 18 3.78v12.44c0 .98-.8 1.78-1.78 1.78H3.78C2.8 18 2 17.2 2 16.22V3.78C2 2.8 2.8 2 3.78 2zM11 9h3v2h-3v3H9v-3H6V9h3V6h2v3z"
+        />
+    </svg>
+`;
+
 /**
  * The template for the {@link @microsoft/fast-foundation#(AccordionItem:class)} component.
  * @public
  */
-export const AccordionItemTemplate = html<AccordionItem>`
-    <template
-        class="${x => (x.expanded ? "expanded" : "")}"
-        slot="item"
-    >
-        <div
-            class="heading"
-            part="heading"
-            role="heading"
-            aria-level="${x => x.headinglevel}"
+export function createAccordionItemTemplate(
```

Comment:
Wouldn't this be a breaking change? I'm confused why we need to take this approach rather than dropping the SVG into the slot by default.

Reply:
Yeah, I noted that this is a breaking change. And that was the initial approach to just add an SVG to the slot. I can still do that for a quick fix to have default icons. My understanding of the issue is when another library wants to use their own default icons, like fluent UI, they still need to keep adding an SVG to override the FAST version every time they implement the component. If this is not an issue I can take the initial approach.

Reply:
> My understanding of the issue is when another library wants to use their own default icons, like fluent UI, they still need to keep adding an SVG to override the FAST version every time they implement the component.
This is definitely still an issue but I think that's different than this ask. @EisenbergEffect has a PR up which would address this (thought that's being reconciled with all requirements). Definitely appreciate the extra effort here but we'd need to plan for a breaking change like this and I think we'd want to make that change across all components.

For this specifically, I think starting with a default SVG as slotted content would suffice and relieve the immediate pain.

Reply:
OK, I can definitely just add an SVG. 
Thanks!